.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,45 +1,16 @@
 **Link the Issue(s) this Pull Request is related to.**
 One line for each Issue. If merging this Pull Request should auto-close the Issue on that line,
-preface the Issue URL/Number with "Fixes".
-
-Every Pull Request should have at least one Issue linked.
+preface the Issue URL/Number with "Fixes". Every Pull Request should have at least one Issue
+linked.
 
 **Summarize your change.**
 If it's a simple change, a sentence or two is probably sufficient. If it's a complex change,
 adding a list of the three or four main points can be helpful.
 
 It's also helpful to describe **why** you're making this change.
 
-**Checklist**
-You can remove this section before creating your Pull Request.
-
-1. Add a Pull Request title. A descriptive single sentence.
-
-2. Fill in the Pull Request body as described above.
-
-3. Request any additional reviewers.
-
-   Everyone listed in [CODEOWNERS](https://github.com/AcademySoftwareFoundation/OpenCue/blob/master/CODEOWNERS) will be automatically added to the reviewer list, though
-   you only need one Code Owner approval before the Pull Request can be removed.
-
-   Feel free to add other reviewers if you think their input would be valuable!
-
-4. Choose Draft/Regular mode.
-
-   If you feel like your change is ready to be merged or close to that, create a normal Pull
-   Request.
-
-   If you feel like your change isn't ready to be merged -- maybe it's just an idea you have -- but
-   you want feedback anyway, feel free to create a Draft Pull Request and add any needed reviewers
-   (Code Owners will not be automatically added as they are for regular Pull Requests). Draft
-
-   You can convert a Draft Pull Request to a regular Pull Request at any point.
-
-   Draft Pull Requests will still have presubmit checks (such as CI tests) run on them, which can
-   be helpful.
-
-5. Sign the CLA.
+<!--
```

Comment:
Should this be a comment?

Reply:
Yeah, could go either way but I figured it's one less thing to delete. The link isn't very useful when it's a comment I suppose.

Reply:
Ok, I got you now. Works either way for me.

Reply:
Sounds good. That markdown link wasn't doing much good in the comment where it will never be rendered as an actual link, so I replaced it with just the URL to the HTML contributing guide.